### PR TITLE
remove hint about excluding node_modules

### PR DESCRIPTION
**Pull request by atomfrede**
This PR removes the hint of excluding `node_modules` from indexing in idea as this prevents code completion etc. in frontend files. See https://github.com/jhipster/generator-jhipster/issues/10860 for details

### CODE REVIEW

**Comment by pascalgrimaud** [comment]
We need to be carefull about that, as not excluding `node_modules`  will slow down the demo / talk we can do.

Then, if you can develop the front part with idea, it means you have a license.
So for our users who use Community Edition of idea (which is the majority I think), it is not possible.

**Comment by atomfrede** [comment]
Good point, you're right, frontend is only ultimate. Let me rephrase the paragraph.

**Comment by pascalgrimaud** [comment]
In my opinion:
- idea ultimate
    - demo -> node_modules should be excluded to speed up idea
    - real project -> depending on your use (some user still prefer VScode for the front)
- idea community -> should always be excluded

**Comment by atomfrede** [comment]
I will close this and rephrase the paragraph about using idea to code frontend code.
